app/components/layout.js

```diff
@@ -41,6 +42,9 @@ class Layout extends React.Component {
     return (
       <div>
         <Navbar {...this.props} connect={this.connect} handleDisconnect={this.disconnect}/>
+        {
+          (this.props.status === 'FAILED') ? <Alert type="danger">{this.props.message}</Alert> : ''
```

Comment:
okie, then please add a card in trello. we need some way to track technical debt.